src/AbstractServer.php

```diff
@@ -111,29 +100,28 @@ protected function _buildSignature(Reflection\AbstractFunction $reflection, $cla
     /**
      * Dispatch method
      *
-     * @deprecated Since 2.7.0; method will be renamed to remove underscore
-     *     prefix in 3.0.
-     *
      * @return mixed
      * @throws ReflectionException
      */
-    // @codingStandardsIgnoreStart
-    protected function _dispatch(Method\Definition $invokable, array $params)
+    protected function dispatch(Method\Definition $invokable, array $params)
     {
-    // @codingStandardsIgnoreEnd
         $callback = $invokable->getCallback();
         $type     = $callback->getType();
 
         if ('function' === $type) {
             $function = $callback->getFunction();
+            // phpcs:disable
             return call_user_func_array($function, $params);
+            // phpcs:enable
```

Comment:
```suggestion
            return $function(...$params);
```

We're bumping the PHP version. We can use the fact that there is now uniform callable syntax and the splat operator. :)

Reply:
This discussion was held previously and we agreed on introducing callables and Iterators separately. Is that okay with you?

If it helps, I can provide a list of issues with planned changes/features. These include among the two mentioned above static analysis, bump to PHP 7.4 (or at least 7.3) and typed properties.